Makefile

```diff
@@ -2,3 +2,10 @@
 run:
 	cd $(PROJECT_DIR) && AUTO_APPROVE="-auto-approve" make
 
+summary:
+	@echo "zero-aws-eks-stack:"
+	@echo "- Repository URL: ${REPOSITORY}"
+	@echo "- To see your kubernetes clusters, run: 'kubectl config get-contexts'"
+	@echo "- To switch to a cluster, use the NAME from the previous command in 'kubectl config use-context NAME'"
```

Comment:
Actually why don't we just show them the correct cluster name here?

Reply:
You want to see separate `kubectl config use-context`'s for each environment that is applied?

Reply:
Yeah